lib/buffer-search.coffee

```diff
@@ -58,6 +59,17 @@ class BufferSearch
       ResultsMarkerLayersByEditor.set(editor, layer)
     layer
 
+  checkSanity: (findPattern) ->
```

Comment:
Yeah, that's a valid approach. We tend to name things such so that they describe the *current* behavior with good specificity, and then rename them later, if and when their behavior changes.